modules/serverless-logic-eventing-system-config-precedence-rule.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+// * serverless-logic/serverless-logic-workflow-eventing-system
+
+:_mod-docs-content-type: REFERENCE
+[id="serverless-logic-eventing-system-config-precedence-rule_{context}"]
+= Eventing system configuration precedence rules
+
+The {ServerlessLogicOperatorName} follows a defined order of precedence to determine the eventing system configuration for a workflow. 
+
+The rules are as follows:
+
+1. If the workflow has a defined eventing system using either workflow-scoped outgoing or incoming eventing system configurations, this configuration takes priority and applies to the workflow.
```

Comment:
Do the rules need to be followed in a particular order? If not, consider bullet points.

Reply:
Yes, they need to be followed in a particular order.